torch/csrc/jit/codegen/cuda/kernel_db/test/test_data/kernel_db_for_query_test/kernel_0.cu

```diff
@@ -0,0 +1,26 @@
+__global__ void kernel1(Tensor<float, 3> T0, Tensor<float, 3> T1, Tensor<float, 3> T2) {
```

Comment:
IIUC, we don't really need a real kernel_0.cu / kernel_0.cubin. Since they are only used for kernel_db.query tests and we are not using the saved kernel. i.e. I don't think we need to attach valid code here :laughing: something like a `dummy_code` would be good enough.